model/version.go

```diff
@@ -142,11 +142,21 @@ func (v *Version) UpdateStatus(newStatus string) error {
 	if err != nil {
 		return err
 	}
-	v.Status = newStatus
-	event.LogVersionStateChangeEvent(v.Id, newStatus)
+
 	return nil
 }
 
+func (v *Version) UpdateGithubCheckStatus(status string) error {
```

Comment:
Because we didn't used to save version github check status, I believe that in the event that tasks/builds had been restarted, we would re-hit this block and re-trigger Github check updates:
```
if checkGithubStatus && activeBuilds > 0 && buildsWithGithubCheckTasksComplete >= activeBuilds {
		event.LogVersionGithubCheckFinishedEvent(versionId, githubCheckStatusFromTasks)
	}
```
I see the benefit of storing this information in the DB given the way you've structured all of this, but I believe we should now consider re-setting GithubCheckStatus during status changes (i.e. restarts). I know you're leaving in like a minute; if this is too big an ask we can make a separate ticket for it, I just didn't want to forget it.

Reply:
I removed the version's GithubCheckStatus field in response to a previous comment.